app-admin/fluent-bit/fluent-bit-4.0.3.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Fast Log processor and Forwarder"
+HOMEPAGE="https://fluentbit.io/"
+SRC_URI="https://github.com/fluent/fluent-bit/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="geoip kafka lua"
+
+DEPEND="
+	acct-group/fluentd
+	acct-user/fluentd
+	app-arch/zstd:=
+	dev-libs/libyaml
+	dev-libs/openssl:=
+	net-dns/c-ares:=
+	net-libs/nghttp2
+	sys-libs/zlib
+	kafka? ( dev-libs/librdkafka )
```

Comment:
This use flag doesn't seem to have any function in this ebuild.

Reply:
Fixed.